src/main/java/uk/gov/hmcts/reform/pdf/service/exception/ExceptionHandling.java

```diff
@@ -24,14 +27,25 @@ public void handleException(Exception exception) {
     }
 
     @ExceptionHandler({
+        InvalidArgumentException.class,
+        MalformedTemplateException.class,
         MissingServletRequestParameterException.class,
         MissingServletRequestPartException.class,
-        InvalidArgumentException.class,
-        MalformedTemplateException.class })
+        ServletRequestBindingException.class })
     @ResponseStatus(code = HttpStatus.BAD_REQUEST)
     @ResponseBody
     public void handleMissingAndMalformedParametersValues(Exception exception) {
         log.error("Input parameters were missing/malformed:", exception);
     }
 
+    @ExceptionHandler(FeignException.class)
+    protected ResponseEntity handleFeignException(FeignException exc) {
```

Comment:
As `ResponseEntity` uses a type parameter I think you need to specify a type or a type wildcard within `<>` brackets, otherwise there may be warnings in the code.